src/inquirer/render/console/_checkbox.py

```diff
@@ -65,14 +65,16 @@ def get_options(self):
             yield choice, selector + " " + symbol, color
 
     def process_input(self, pressed):
+        # Remove after merge #83
+        key.SHIFT_TAB = "\x1b\x5b\x5a"
```

Comment:
Instead of adding a code while a PR is not merged, I would focus in having the dependency updated. This is creating some double work for you, me and other people working on python-inquirer.

Reply:
Last update in readchar 4 feb 2022. I dont now, how many time need for merge it.

Reply:
@magmax do you think we can have a new version with magmax/python-readchar#83 in forseable future?

Reply:
Now https://github.com/magmax/python-readchar/pull/83 is part  of https://github.com/magmax/python-readchar/pull/79 , но в случае успешного ответа, я переоткрою тот МР

Reply:
this is already merged and in pre-release state